src/runner.ts

```diff
@@ -229,6 +229,12 @@ export class StaticAnalysisParserRunner {
                     vulnerabilityDetailDescription = vulnerabilityDetailDescription.slice(0, 1997) + "...";
                 }
 
+                if (result.partialFingerprints.violType == "FlowViol") {
```

Comment:
We also need to consider the situation where the description length exceeds 2000